adaptive_navigation/example/.firebaserc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Are you intentionally including the `.firebaserc` and `firebase.json` files?

Reply:
yes, even though I am currently the only one that can deploy the demo, but I can add collaborators as this package becomes more official